device/colibri.go

```diff
@@ -0,0 +1,314 @@
+package device
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/xshellinc/esp-flasher/serialport"
+	"github.com/xshellinc/go-serial"
+	"github.com/xshellinc/iotit/device/config"
+	"github.com/xshellinc/iotit/workstation"
+	"github.com/xshellinc/tools/dialogs"
+	"github.com/xshellinc/tools/lib/help"
+)
+
+const (
+	COLIBRI = "colibri"
```

Comment:
__[golint]__
* [golint] exported const COLIBRI should have comment (or a comment on this block) or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->